### DISCUSSION TITLE
Why is the Silk.NET discord server now Egypt.NET?

### DISCUSSION THREAD

**Discussion by dfkeenan** [dotnet/Silk.NET#1379] [General] [answered]
Hi,

Did the discord server get taken over or something?

**Comment by Beyley** [comment, marked as answer]
As with all our important announcements on the Discord, we make them on the first day of April

**Comment by dfkeenan** [comment]
Only fools go on the internet on the 1st of April. 😄 

Time to climb under a rock.
